compiler/resolution/virtualDispatch.cpp

```diff
@@ -75,8 +75,8 @@ static void addToVirtualMaps   (FnSymbol* pfn, AggregateType* ct);
 static void collectMethodsForVirtualMaps(Vec<FnSymbol*>& methods,
                                          Type*           ct,
                                          FnSymbol*       pfn);
-static void collectInstantiatedAggregateTypes(Vec<Type*>& icts,
-                                              Type*       ct);
+static void collectInstantiatedAggregateTypes(Vec<AggregateType*>& icts,
```

Comment:
Is this a good opportunity to

a) convert Vec<t> to std::vector<t>

b) Follow the common convention that formals that are effectively output
values should be grouped as the final formals rather than being first.

Not wedded to either of these but I know I would take this opportunity.